packages/ext-gen/ext-gen.js

```diff
@@ -613,31 +648,31 @@ async function stepCreate() {
     else {
       if (require('os').platform() == 'win32') {
         //args = ['install','-s','>','NUL']
-        args = ['install','-s']
+        args = ['install', '-s']
       }
       else {
         //args = ['install','-s','>','/dev/null']
-        args = ['install','-s']
+        args = ['install', '-s']
       }
     }
-    let options = {stdio: 'inherit', encoding: 'utf-8'}
+    let options = { stdio: 'inherit', encoding: 'utf-8' }
     console.log(`${app} npm ${args.toString().replace(/,/g, " ")} started for ${answers['packageName']}`)
     await util.spawnPromise(command, args, options, substrings);
     console.log(`${app} npm ${args.toString().replace(/,/g, " ")} completed for ${answers['packageName']}`)
-  } catch(err) {
+  } catch (err) {
     console.log(boldRed('Error in npm install: ' + err));
   }
 
-  var frameworkPath = path.join(destDir, 'node_modules', npmScope, 'ext', 'package.json');
```

Comment:
@n0wfal @sathishcel  what is the reason npm scope was removed here?  

Can you add proper comments...  nothying will impact existing functionality?

Reply:
change done by @beeonlink (Shanmugam)  package.json is not available inside the node_modules directory instead it is available in the root of the application. so it is removed. 
Akshay has tested in the PR level(using local dev changes) we can test more after PR merge thoroughly.
I will keep note of this change until we complete 100% testing.